HTML+CSS+JS/Navigation-Bar/Navbar-Bubble-Animation/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Navbar Animation</title>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<nav>
+		<ul>
+			<div class="menu">

```

Comment:
formatting is a bit off here @Pranav016 ,otherwise rest seems fine

Reply:
Thanks for pointing out 👍 
Our prettier bot does some auto formatting when a PR is merged so I think this will be taken care by it.